.github/workflows/push-bioconductor-devel.yml

```
@@ -0,0 +1,32 @@
+        uses: webfactory/ssh-agent@v0.9.0
+        with:
+          ssh-private-key: ${{ secrets.BIOC_SSH_PRIVATE_KEY }}
+      - name: Add SSH host key
+        run: |
+          ssh-keyscan -H git.bioconductor.org >> ~/.ssh/known_hosts
+      - name: Add Bioconductor remote
+        run: |
+          git remote add upstream git@git.bioconductor.org:packages/MSstatsBioNet.git
+      - name: Push to Bioconductor
+        run: |
+          git push upstream devel
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)